Modix.Bot/Modules/ReplModule.cs

```diff
@@ -150,19 +147,20 @@ private async Task<EmbedBuilder> BuildEmbedAsync(IGuildUser guildUser, Result pa
         {
             var returnValue = parsedResult.ReturnValue?.ToString() ?? " ";
             var consoleOut = parsedResult.ConsoleOut;
-            var status = string.IsNullOrEmpty(parsedResult.Exception) ? "Success" : "Failure";
+            var exception = !string.IsNullOrEmpty(parsedResult.Exception);
```

Comment:
The name `exception` gives the impression that the variable is the exception itself. Maybe `hasException` instead?

Reply:
Sure